### PR TITLE
[IMPROVE] Drag drop attachments

### PR DESCRIPTION
**Pull request by RahulNavneeth**
# Description
This pull request adds the ability to drag and drop attachments directly into the chat window, making it easier and more intuitive for users to share files with one another.

## Changes Made
The following changes have been made to implement this feature:

Added drag and drop event listeners to the chat window.
Implemented logic to handle file uploads when files are dropped into the chat window.
Added a UI component to display the uploaded file in the chat window.

BEFORE:

https://user-images.githubusercontent.com/50259284/221878096-85ef089c-47ba-45c0-83cc-aba4899e43fb.mp4

AFTER:

https://user-images.githubusercontent.com/50259284/221878205-e269e04d-fe38-472b-9fb5-c29a671f3c6b.mp4

### CODE REVIEW

**Comment by RahulNavneeth** [comment]
hey @sidmohanty11 , Can you kindly review this pull request. Thanks 😄

File: src/components/ChatBody/ChatBody.js [hunk 1]
```diff
@@ -60,15 +61,53 @@ const ChatBody = ({ height, anonymousMode, showRoles, GOOGLE_CLIENT_ID }) => {
...
+      onDragLeave={handleDragLeave}
       className={styles.container}
       height={height}
     >
+      {onDrag ? (
+        <Box
+          onDrop={(e) => handleDragDrop(e)}
+          className={styles.drag_component}
+        >
+          DRAG AND DROP AREA
```
Comment (sidmohanty11) [inline src/components/ChatBody/ChatBody.js:108]: can we please change it to - `Drop to upload file` - same as RC

**Review by sidmohanty11 — CHANGES_REQUESTED** [review]
This is amazing @RahulNavneeth ! Thanks!

File: src/components/ChatBody/ChatBody.js (hunk 1 above)
Comment (RahulNavneeth) [reply src/components/ChatBody/ChatBody.js:108]: Sure sir 👍

Comment (RahulNavneeth) [reply src/components/ChatBody/ChatBody.js]: Done 👍

**Review by sidmohanty11 — APPROVED** [review]
LGTM! Thanks @RahulNavneeth

**Comment by RahulNavneeth** [comment]
Thank you so much, @sidmohanty11 😄

**Pull request merged**